packages/arb-node-core/nodehealth/nodehealth.go

```diff
@@ -96,10 +99,24 @@ func updateConfig(config *configStruct, logMessage Log) {
 	defer config.mu.Unlock()
 	//Load the configuration message into the config struct
 	if logMessage.Var == "openethereumHealthcheckRPC" {
-		config.openethereumHealthcheckRPC = logMessage.ValStr
+		u, err := url.Parse(logMessage.ValStr)
+		if err != nil {
+			return
+		}
+		config.openethereumHealthcheckRPC = u.Hostname()
```

Comment:
Not sure why we're stripping out the port here and delivering the hostname and port separately?

Reply:
The healthcheck is on a separate port from the RPC endpoint that is passed in via a command line argument.  This approach gives us configurability for the default healthcheck port inside the nodehealth file versus having to update it across the code base.

Reply:
Ah cool